content/influxdb/cloud-dedicated/query-data/execute-queries/optimize-queries.md

```diff
@@ -29,7 +31,25 @@ Use the following tools to help you identify performance bottlenecks and trouble
   - [Inspect Flight response headers](#inspect-flight-response-headers)
 - [Retrieve query information](#retrieve-query-information)
 
-<!-- /TOC -->
+
+## Strategies for improving query performance
+
+A query may be slow due to the following reasons:
+
+- It queries a large time-range of data.
+- It includes intensive operations, such as `ORDER BY` on large amounts of data.
+- The query plan isn't optimal--for example, applying the same sort (`ORDER BY`) to already sorted data.
```

Comment:
Usually, user cannot control or change this. Our querier's optimization has to be smart enough to know this and avoid resort. If you want to mention this, maybe put it in the `explain` session where you show users how to read and query plan and id they see the already-sorted data get resorted, report that to us

Reply:
I would be tempted to use slightly different language as well.

A query being 'slow' is relative - what are their expectations, are they reasonable, are the compared to other technologies.

We are better off explaining that query response times are couple to, or proportional to, the time range of the query (wider time ranges will take longer to process) and the complexity of the query operation to be performed.